### PR TITLE
adding link to explanation of cpu limits or requests in the onboarding doc

### PR DESCRIPTION
**Pull request by mubariskhan96**

### CODE REVIEW

File: docs/content/cluster-scope/quotas.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ When requesting a namespace, you can choose from 1 of 4 tier options for your re
 | Medium  |     2     |   8Gi  |   40Gi  |
 | Large   |     4     |  16Gi  |   80Gi  |
 
+To understand the difference between cpu limits/requests, please refer [this][k8s-doc].
```
Comment (durandom) [inline docs/content/cluster-scope/quotas.md:12]:
```suggestion
To understand the difference between cpu limits/requests, please refer to the [kubernetes concepts][k8s-doc]. 
```

**Review by durandom — CHANGES_REQUESTED** [review]
suggested to be more descriptive

File: docs/content/cluster-scope/quotas.md (hunk 1 above)
Comment (HumairAK) [inline docs/content/cluster-scope/quotas.md:12]: Change `please refer [this][k8s-doc]` to `please refer to [this][k8s-doc]`

Comment (HumairAK) [inline docs/content/cluster-scope/quotas.md:12]: Change `cpu` to `cpu and memory`

Comment (mubariskhan96) [reply docs/content/cluster-scope/quotas.md]: done

Comment (mubariskhan96) [reply docs/content/cluster-scope/quotas.md]: done

**Comment by 4n4nd** [comment]
/ok-to-test

Can you squash your commits to one?

**Comment by mubariskhan96** [comment]
> /ok-to-test
> 
> Can you squash your commits to one?

Yes, I'm looking into it. But there have been some changes pushed since i created this PR and its creating some issues with my git. Looking at resolving this.

**Comment by mubariskhan96** [comment]
Actually, I think I need some help with squashing the commits into one. Its not working for me. Would appreciate if someone could get on a quick call with me to help me with it. Thanks!

File: docs/content/cluster-scope/quotas.md (hunk 1 above)
Comment (mubariskhan96) [reply docs/content/cluster-scope/quotas.md]: incorporated changes as suggested by @HumairAK

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**